services/cell_logger/cell_logger.py

```diff
@@ -81,6 +82,11 @@ def write_to_db(timestamp, signal, cell_tech):
 
     print("Finished inserting cell signal power data for timestamp ", timestamp)
 
+
+#Create a metric to track cell signal
+
+signal_gauge = Gauge('received_cell_signal_power', 'Received cell signal power')
```

Comment:
I would change the name of this metric. Things like to break metrics names by `_` so the first word because a "group name" for a set of metrics.

Maybe `cell_receive_power` ? Then we can have things like `cell_access_type` , `cell_sms_rx`, etc. in the future.

Reply:
Maybe even, `mm_` rather then `cell_`?

Reply:
I left `avena_cell` to indicate that this cell signal comes from a component of the Avena stack as discussed.